lib/shinseibank/cli/subcommand.rb

```diff
@@ -0,0 +1,17 @@
+class ShinseiBank
+  class CLI < Thor
+    class Subcommand < Thor
+      class_option :credentials, required: true, type: :string
+
+      private
+
+        def shinseibank
+          @_shinseibank ||= ShinseiBank.new
```

Comment:
Heh, yeah, it's become a habit at work: instance variables used as "cache", not supposed to be used anywhere else, are prefixed by an underscore. I don't have to use that "private rule" here, but I think it's nice.